### DISCUSSION TITLE
Add Web Assembly Package Support in GitHub Package

### DISCUSSION THREAD

**Discussion by beingminimal** [community/community#49375] [Packages]
### Select Topic Area

Product Feedback

### Body

Hi Community!

As we all know web assembly is the future.
And to support web assembly we need to support its package management in github.
So please add web assembly package management support in [GitHub Packages](https://github.com/features/packages).

Here i have attached few important links for references.
https://wasi.dev/
https://wasmer.io/
https://wapm.io/
https://webassembly.org/

Awaiting community support. Thanks

**Comment by beingminimal** [comment]
@otter-computer any update or consideration for this?

**Reply by otter-computer** [reply]
Hey @beingminimal, I don't have anything to report on this yet, but posting this thread is already the best way to get the Packages engineering team to consider this feedback! If I see any movement on this internally I'll be sure to keep you updated.

**Reply by beingminimal** [reply]
@otter-computer okay, but can you raise internal feature request based on this discussion? Please
